NewForce-Cohort-5/tabloidfullstack-spring-of-deception
# Np edit comment

# Description

I added the ability to delete comments on a post. Users should be able to delete their comments only.

## Type of change

[Check Trello Ticket #22](https://trello.com/b/IVGnzwap/spring-of-deception)


# Testing Instructions

1. git add and git commit all changes in your current branch.
2. git checkout main
3. git fetch -a
4. git checkout np-create-comment
5. Navigate to the root directory for this repo (directory-to)/tabloidfullstack-spring-of-deception
6. Start Tabloid.sln
7. Make sure your db is created with seed data (Skip to 9 if this is already done)
8. Run [01_Tabloid_Create_DB.sql](https://github.com/NewForce-Cohort-5/tabloidfullstack-spring-of-deception/blob/main/SQL/01_Tabloid_Create_DB.sql) and [02_Tabloid_Seed_Data.sql](https://github.com/NewForce-Cohort-5/tabloidfullstack-spring-of-deception/blob/main/SQL/02_Tabloid_Seed_Data.sql) queries in Visual Studio
9. Run Query below to add Deactivated UserType to Database
10. ***Run NPM Install to install dependencies***
11. Start the server in Debug Mode F5
12. Log in as admin foo@bar.com
13. Navigate to the posts tab
14. Click on productize turn-key convergence
15. Comments should be below the post
16. If the comment is by the current user there will be a edit button on the comment card
17. If there are no comments with the ability to edit, add a comment first
18. Click on edit comment on whichever comment you would like to delete
19. A window will pop up with current comment values
20. Edit which comment values you would like
21. Click save comment
22. You will now see the edited comment

```sql
INSERT INTO UserType (Name)
Values ('Deactivated')
```


# Checklist:
